### PR TITLE
Feature/topologyspreadcontraint

### PR DESCRIPTION
**Pull request by dschunack**
**Describe what this PR does**

Add the possibility to use topologySpreadConstraints in the helm chart.

**Is there anything that requires special attention?**

NONE

**Related issues:**

#194

### CODE REVIEW

**Comment by JohnStrunk** [comment]
@dschunack Thanks for the PR!
Looks like there's just a line length issue that the linter doesn't like, then it should be good to go.
```
./helm/snapscheduler/README.md:147: MD013 Line length
```

**Comment by dschunack** [comment]
@JohnStrunk I added a new line to fix it :-)

File: helm/snapscheduler/values.yaml [hunk 1]
```diff
@@ -51,6 +51,14 @@ nodeSelector:
 
 tolerations: []
 
+topologySpreadConstraints: []
+  # - maxSkew: 1
+  #   topologyKey: topology.kubernetes.io/zone
+  #   whenUnsatisfiable: DoNotSchedule
+  #   labelSelector:
+  #     matchLabels:
+  #       app.kubernetes.io/instance: snapscheduler
```
Comment (JohnStrunk) [inline helm/snapscheduler/values.yaml:60]:
```suggestion
# - maxSkew: 1
#   topologyKey: topology.kubernetes.io/zone
#   whenUnsatisfiable: DoNotSchedule
#   labelSelector:
#     matchLabels:
#       app.kubernetes.io/instance: snapscheduler
```

It's also possible to run the linter locally via `./.ci-scripts/pre-commit.sh`

**Comment by JohnStrunk** [comment]
DCO didn't like the missing sign-off...
If you pull the branch to get github's commit, then: `git commit -s --amend --no-edit`, it should add your sign-off, then you can `git push --force`

**Comment by dschunack** [comment]
@JohnStrunk i know, i used the resolve Button in the GitHub WebUI. I will fix this tomorrow

**Comment by dschunack** [comment]
@JohnStrunk so, now with sign-off :-)

**Review by JohnStrunk — APPROVED** [review]
Thanks! :tada:

**Comment by dschunack** [comment]
@JohnStrunk Is it possible to release a new version. Without we can't use the new features.

```
helm search repo backube/snapscheduler
NAME                 	CHART VERSION	APP VERSION	DESCRIPTION                                       
backube/snapscheduler	3.0.0        	3.0.0      	An operator to take scheduled snapshots of Kube...
```

**Pull request merged**